conf/machine/include/imx8mm-evk.inc

```diff
@@ -22,6 +22,11 @@ KERNEL_DEVICETREE_append_use-nxp-bsp = " \
 "
 UBOOT_DTB_NAME = "${KERNEL_DEVICETREE_BASENAME}.dtb"
 
+IMX_DEFAULT_BOOTLOADER_use-nxp-bsp = "u-boot-imx"
+IMX_DEFAULT_BOOTLOADER_use-mainline-bsp = "u-boot-imx"
```

Comment:
I might be missing a big picture here, but this line (and the same for `imx8mn`) has caught my attention: why do we enforce the _u-boot-imx_ for mainline BSP?

I believe that current `u-boot-fslc` does have a support for those machines, so why not just use it?

Reply:
Because it is wrong :-) Good catch.